cmd/goplugin-terraform/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"os"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/lyraproj/issue/issue"
+	"github.com/lyraproj/lyra/cmd/goplugin-terraform/terraform"
+)
+
+func init() {
+	if hclog.DefaultOptions.Level <= hclog.Debug {
```

Comment:
This statement isn't triggered correctly unless it's placed after the configuration of `hclog.DefaultOptions`.

Reply:
👍